lib/monitor.js

```diff
@@ -3,10 +3,12 @@
 var Events = require('events');
 var Http = require('http');
 var Https = require('https');
-var Os = require('os');
 var Hoek = require('hoek');
 var Items = require('items');
+var Os = require('os');
```

Comment:
Os was before Hoek to include node core modules first, then 3rd party, then local.  This needs to be added to the styleguide.